google/cloud/dns/client.py

```diff
@@ -91,14 +91,23 @@ def quotas(self):
 
         :rtype: mapping
         :returns: keys for the mapping correspond to those of the ``quota``
-                  sub-mapping of the project resource.
+                  sub-mapping of the project resource. ``kind`` is stripped
+                  from the results.
         """
         path = "/projects/%s" % (self.project,)
         resp = self._connection.api_request(method="GET", path=path)
 
-        return {
-            key: int(value) for key, value in resp["quota"].items() if key != "kind"
-        }
+        quotas = resp["quota"]
```

Comment:
Is there an advantage to getting the quotas and popping kind over a list comprehension?  It appears the issue was the type conversion to int?

Reply:
This method started failing because there's a new key `whitelistedKeySpecs` in `quota`. str -> int conversion failed since it's a list

https://cloud.google.com/dns/docs/reference/v1/projects#resource
```
  "quota": {
    "kind": "dns#quota",
    "managedZones": integer,
    "rrsetsPerManagedZone": integer,
    "rrsetAdditionsPerChange": integer,
    "rrsetDeletionsPerChange": integer,
    "totalRrdataSizePerChange": integer,
    "resourceRecordsPerRrset": integer,
    "dnsKeysPerManagedZone": integer,
    "whitelistedKeySpecs": [
      {
        "kind": "dns#dnsKeySpec",
        "keyType": string,
        "algorithm": string,
        "keyLength": unsigned integer
      }
    ],
    "networksPerManagedZone": integer,
    "managedZonesPerNetwork": integer,
    "policies": integer,
    "networksPerPolicy": integer,
    "targetNameServersPerPolicy": integer,
    "targetNameServersPerManagedZone": integer
  }

```

Each entry`whitelistedKeySpecs` also has a 'kind' that needs to be stripped. I found it easier to pop than to edit the existing list comprehension.